extra/2-piping.js

```diff
@@ -16,30 +16,31 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+return a + b;
 }
 
-function multiply() {
-
+function multiply( a, b) {
+return a * b;
 }
 
-function format() {
+function format(number) {
+  return `£${number}`;
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
```

Comment:
The task has asked you to comment here why you think this is bad code :).

Reply:
I forgot to add comments. i have added now. Thank you so much for pointing it out.